.dockerignore

```diff
@@ -0,0 +1,14 @@
+.deployment/
```

Comment:
Is this file really necessary? What docker command would automatically do what with all the files mentioned here otherwise? Seems like the Dockerfile already copies only very specific files and folders?

Reply:
Yes, not every file is used in the `Dockerfile`. I'm pretty sure, they will still be uploaded to the Docker daemon if not ignored. Running a container build will upload two files: the Dockerfile + the build context. The build context is a folder (here the root project folder) excluding every file from `.dockerignore`.